package.json

```diff
@@ -20,7 +20,11 @@
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.18.9",
+    "@emotion/react": "^11.10.5",
```

Comment:
nit: do we really need these?

Reply:
nope - they were in the material ui setup instructions but weren't actually necessary. removed them